### ISSUE TITLE
fake Rocket Fuels' value

### ISSUE THREAD

**Issue by InTheBackOf** [GTNewHorizons/GT-New-Horizons-Modpack#9684]
### Your Pack Version

2.1.2.2

### Your Server

sp

### Type of Server

Single Player

### Your Expectation

Univarsal Chemial Fuel Engine can use rocket fuel in GTNH2.1.2.2. The result shows it has enough power for a normal LuV tier's player. But the real test result ( 60mB/t CN3H7O3 rocket fuel + 3mB/t combustion promoter ) is only has about 166Keu/t instead of 500Keu/t from NEI data.
Other rocket fuels by Alk. have the same problem too. 

Other usable fuels like HOG in Univarsal Chemial Fuel Engine are work rightly.

### The Reality

![RP-1显示热值](https://user-images.githubusercontent.com/38945243/152643882-09c31ef8-4efd-49d5-bd90-a26ff08328f3.jpg)
![绿色燃料显示热值](https://user-images.githubusercontent.com/38945243/152643900-2cee4d09-984a-4de7-b77d-1683767b60b2.jpg)
![密集肼显示热值](https://user-images.githubusercontent.com/38945243/152643905-b80a8db0-460c-4caa-bf3a-78020a593c75.jpg)
![紫色燃料显示热值](https://user-images.githubusercontent.com/38945243/152643910-9e9d8046-3df3-4155-903b-ed88d15d374c.jpg)

the code from [https://github.com/GTNewHorizons/GTplusplus/blob/master/src/main/java/gtPlusPlus/core/item/chemistry/RocketFuels.java#L308-L356](url) shows all of them only have 1/3 vaule.

### Your Proposal

make them show the reality.and let rocket fuel engine is not a toy.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by bombcar** [comment]
Do they need oxygen or other boosting?

**Comment by InTheBackOf** [comment]
> 

UCE must use combustion promoter, or any engery will not output. More combustion promoter will take higher efficiency.

**Comment by Steelux8** [comment]
Interesting, I did not know about this and it's not documented anywhere. I only tested the UCE with HOG to test the efficiency values at different fuel:promoter ratios.

To answer this, I'll say that Alkalus has said that the Rocket Fuel Engine should be the best at burning rocket fuel, because it's the dedicated multi for it, and I also agree that the UCE should not be better than it. Secondly, the RFE is different in this aspect because it works in a different way, and has "free fuel ticks", which is to say, it produces EU constantly but only burns fuel every other second. This, amongst other things, makes it very efficient for the rocket fuel, and I believe it makes those fuels measure to the value shown on NEI.
